docs/maui/views/Expander.md

```diff
@@ -0,0 +1,157 @@
+---
+title: Expander - .NET MAUI Community Toolkit
+author: VladislavAntonyuk
+description: "The Expander control provides an expandable container to host any content."
+ms.date: 11/08/2022
+---
+
+# Expander
+
+The `Expander` control provides an expandable container to host any content. The control has two main properties to store your content:
+
+## Header
+
+This `Header` property can be provided with any view to allow for full customization. The `Header` will always be visible and interacting with it (clicking or tapping) will show/collapse the `Content`.
+
+> [!NOTE]
+> It is not recommended to place controls inside the header that allow user interaction.
+
+## Content
+
+This is the main content that will show when the `Header` property is interacted with it (clicked or tapped) or the `IsExpanded` property is modified.
+
+> [!NOTE]
+> `Expander` is not supported inside `CollectionView` or `ListView`
+
+![Screenshot of an Expander in collapsed and expanded states](../images/views/Expander.gif "Expander on Android")
```

Comment:
@brminnick and @VladislavAntonyuk where are we with `ListView` and `CollectionView` on Android? Did we need to rework this sample to not show something we are recommending against doing?

Reply:
I added 2 images. 
The first image contains ListView (it is not used right now), and the second one doesn't contain ListView.